include/ews/ews.hpp

```diff
@@ -17072,6 +17146,44 @@ class basic_service final
         return create_item_impl(the_message, disposition, folder);
     }
 
+    //! \brief Creates new messages in the Exchange store.
+    //!
+    //! Creates new messages and, depending of the chosen message
+    //! disposition, sends them to the recipients.
+    //!
+    //! Note that if you pass message_disposition::send_only or
+    //! message_disposition::send_and_save_copy this function always
+    //! returns an invalid item id because Exchange does not include
+    //! the item identifier in the response. A common workaround for this
+    //! would be to create the item with message_disposition::save_only, get
+    //! the item identifier, and then use the send_item to send the message.
+    //!
+    //! \return A vector of the item ids of the saved messages when
+    //! message_disposition::save_only was given; otherwise an invalid item
+    //! id.
+    std::vector<item_id> create_item(const std::vector<message>& messages,
+                                     ews::message_disposition disposition)
```

Comment:
Two things here:

* `ews::` namespace qualifier is redundant and not needed here. Remove.
* docs: Does this mean that I get a vector full of invalid item_ids when disposition is `send_only` or `send_and_save_copy`. Clarify.